Server/JokesOnYou.Web.Api/Services/Interfaces/IJokesService.cs

```diff
@@ -6,6 +6,7 @@ namespace JokesOnYou.Web.Api.Services.Interfaces
 {
     public interface IJokesService
     {
+        Task UpdateJoke(JokeUpdateDto jokeUpdateDto);
```

Comment:
I think its a good idea to return the updated joke, just like we do when we create it